cypress/e2e/APIRequests.cy.ts

```diff
@@ -505,9 +540,11 @@ describe('Partially successful API query requests', () => {
   it('Shows a warning for nodes that failed to return any results', () => {
     cy.get('[data-cy="submit-query-button"]').click();
     cy.wait('@call');
-    cy.get('.notistack-SnackbarContainer')
-      .find('.notistack-MuiContent-warning')
-      .should('contain', 'DidNotWorkNode');
+    cy.get("[data-cy='navbar']").find('svg[data-testid="NotificationsIcon"]').should('exist');
```

Comment:
Feel free to add a `data-cy` to the notification button so you don't have to access it through the navbar every time.